recipe/meta.yaml

```diff
@@ -1,32 +1,33 @@
 {% set name = "sacrebleu" %}
-{% set version = "2.0.0" %}
+{% set version = "2.4.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 51fb69b6683f1b9999cd180143bb6b21d7841744537c9aab235cfe676550f0cf
+  url: https://github.com/mjpost/sacrebleu/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
I'm curious the reason for switching the source to the github url?

Reply:
I just noticed your comment in the description, but maybe updating to `2.4.3` will remove the issue as it seems to be listed [here](https://inspector.pypi.io/project/sacrebleu/)